scripts/remove/jdownloader.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# JDownloader remover for swizzin
+# Author: Aethaeran
+
+# TODO: Not sure this import is necessary
+# Import swizzin utils
+. /etc/swizzin/sources/functions/utils
+
+# this will check for all swizzin users, and iterate and remove all JDownloader installations.
+users=("$(_get_user_list)")
+# The following line cannot follow SC2068 because it will cause the list to become a string.
+# shellcheck disable=SC2068
+for user in ${users[@]}; do
+    echo_progress_start "Removing JDownloader for $user..."
+    systemctl disable -q --now jdownloader@"$user"
+    # TODO: Do a check to see if they want to purge their JDownloader configurations.
```

Comment:
we don't currently have purge functionality built into swizzin. It is being looked at for our v4 milestone as a box flag. If you did want to include this option, it might be a good idea to push into its own purge function ;)

Reply:
Was assuming it would be purged on removal, or that I could add a simple yes/no here.
deluser purges all configurations, so I guess I just assumed a feature removal would as well.

Reply:
I've now added the yes/no prompts for configuration purging on removal.

At the very least I'd like to leave it this way until testing is thoroughly completed, then just perhaps move that logic and operation to a purge_jdownloader function.